lib/src/chrome.dart

```diff
@@ -119,16 +122,23 @@ class Chrome {
     // Wait until the DevTools are listening before trying to connect.
     final errorLines = <String>[];
     try {
-      await process.stderr
+      final stderr = process.stderr.asBroadcastStream();
+      stderr
+          .transform(utf8.decoder)
+          .transform(const LineSplitter())
+          .listen(_logger.fine);
+
+      await stderr
           .transform(utf8.decoder)
           .transform(const LineSplitter())
           .firstWhere((line) {
         errorLines.add(line);
         return line.startsWith('DevTools listening');
       }).timeout(const Duration(seconds: 60));
-    } catch (_) {
+    } catch (e, s) {
```

Comment:
Is there a `TimeOut` exception or similar that we could catch explicitly and report back on.

Is 60s too long? Shouldn't it by pretty quick?

Reply:
> Is 60s too long? Shouldn't it by pretty quick?

Maybe, but I am afraid to change that as this code is used in many slow CI systems... We keep finding timing failures (just like this PR shows).

Reply:
> Is there a TimeOut exception or similar that we could catch explicitly and report back on.

Will try that, thanks!

Reply:
Done!